airbyte-integrations/connectors/source-google-analytics-data-api/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
This file seems identical to `integration_tests/configured_catalog.json` except from the missing `cohort_report` stream. I don't see this file reference anywhere. Why do we need this file? Can we delete it?

The same question applies to `airbyte-integrations/connectors/source-google-analytics-data-api/sample_files/sample_config.json`